lms/djangoapps/teams/tests/test_api.py

```diff
@@ -190,6 +190,7 @@ def test_anonymous_user_ids_for_team(self):
         A learner should be able to get the anonymous user IDs of their team members
         """
         team_anonymous_user_ids = teams_api.anonymous_user_ids_for_team(self.user1, self.team1)
+        self.assertTrue(AnonymousUserId.objects.get(user=self.user1, course_id=self.team1.course_id))
```

Comment:
Slightly cleaner?:
`self.assertTrue(AnonymousUserId.objects.filter(user=self.user1, course_id=self.team1.course_id).exists())`

Reply:
I don't even think you need the assert. get raises DoesNotExist, so if it doesn't raise that, you're good. That said, asserting an object is true is a bit ... incorrect, sure it's truthy, but it's better to use, like, isNotNone or something like that